apps/api/src/auth/service/auth.service.ts

```diff
@@ -84,4 +85,14 @@ export class AuthService {
       token: await this.jwt.signAsync({ id: user.id })
     }
   }
+
+  @Cron(CronExpression.EVERY_HOUR)
+  async cleanUpExpiredOtps() {
+    try {
+      await this.authRepository.deleteExpiredOtps()
```

Comment:
don't put this on a blocking call. I believe that won't be necessary

Reply:
Yeah, you can always chain up the logger to be explicit about errors and success messages

Reply:
> don't put this on a blocking call. I believe that won't be necessary

I think it is fine to put it here as this will not block any flow and once the internal work is done we will log the message synchronously

Reply:
ummm, yeah you are right. we are not calling it explicitly anywhere